src/model/defineRelationalProperties.ts

```diff
@@ -36,6 +36,9 @@ export function defineRelationalProperties(
         `defining relational property "${entity.__type}.${property}"`,
         relation,
       )
+
+      if (!(property in initialValues)) return properties
```

Comment:
This changes the established behavior, but I think it's a good change. Previously, it was an explicit decision to throw on missing relational properties, but looking at it now, I don't think that's necessary. You can attach the relational value later, which also makes those properties optional (just as any other entity properties).

In other words, where base values use value getters to populate created entities, relational properties must be explicitly set (they don't have a getter, instead they have a relational node produced by `oneOf`/`manyOf`). We should document this in README.

Reply:
do you think that something like this could be useful 

```diff
diff --git a/README.md b/README.md
index b9877fa..e04c19d 100644
--- a/README.md
+++ b/README.md
@@ -458,6 +458,8 @@ db.user.findFirst({
 
 Defining relationships is a way for one model to reference another. Models are flat by design, so if you wish to have a property that equals an object or an array, you create a relationship of the proper type to another existing model.
 
+All relations are optional as any other property specified in the model. You can create an entity and add relations later using the update API.
+
 #### One-to-One
 
 ```

Reply:
I think we need to elaborate on all the model properties being optional in the [create()](https://github.com/mswjs/data#create) section of the README. It also misses the example of calling `.create()` with initial values, which I think would be nice to showcase.

Reply:
I've pushed this addition to the README to this feature branch.